packages/gitgraph-react/src/BranchLabel.tsx

```diff
@@ -4,8 +4,6 @@ import { Branch, Commit } from "@gitgraph/core";
 interface Props {
   branch: Branch<React.ReactElement<SVGElement>>;
   commit: Commit<React.ReactElement<SVGElement>>;
-  x: number;
-  y: number;
```

Comment:
To simplify and make it scalable, we now compute x/y at a single place, after component updates. 

Thus, this is not necessary.